myip.go

```diff
@@ -49,3 +50,16 @@ func GetIP() (net.IP, error) {
 
 	return nil, fmt.Errorf("Unable to find IP")
 }
+
+// ExternalIP returns your public IP.
+func ExternalIP() (string, error) {
```

Comment:
The function name and comments don't really explain how this is different from the other function and makes it very confusing. Perhaps consider including the name of the service and/or the full url in the comments and changing the function name to reflect the service.